.ent/ent-prj

```diff
@@ -0,0 +1,2 @@
+# ENT-PRJ / 2022-01-27T21:35:08+0000
```

Comment:
@nshaw @w-caffiero-entando if I understood well the problem, pipelines are failing because we have both `ent-prj `and` pom.xml` files in the sources, version seems to be resolved in this file instead of `pom.xml`

Reply:
@cecchisandrone I'm pretty sure ent-prj must be there for proper ent support.  And on ppl v1.2 I think there's a way to specify the type of the project in the pr.yaml to avoid that conflict but my knowledge may be out of date. Setting as secret to mvn may be needed plus this change in pr.yaml?
ENTANDO_OPT_PROJECT_TYPE: "${{ secrets.ENTANDO_OPT_PROJECT_TYPE }}"